<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectThrashold</class>
 <widget class="QWidget" name="selectThrashold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Min H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_min_h">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Max H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_max_h">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Min S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_min_s">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Max S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_max_s">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Min V</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_min_v">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Max V</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="s_max_v">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>213</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>s_min_h</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_max_h</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_min_s</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_13</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_max_s</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_min_v</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_17</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>s_max_v</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_15</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
